src/commands/games/blackjack.ts

```diff
@@ -190,13 +182,17 @@ class BlackjackCommand extends BaseCommand {
     adjustCoinBalanceByUserId(playerId, balanceChange, UserCoinEvent.Blackjack);
   }
 
-  async onRun(message: CommandoMessage, args: { bet: number }): Promise<Message> {
-    const { bet } = args;
+  async messageRun(message: Message, args: Args): Promise<Message> {
+    const bet = args.finished
+      ? DEFAULT_BET
+      : await args.rest('integer').catch(() => 'please enter a valid bet amount.');
+    if (typeof bet === 'string') return message.reply(bet);
+
     const { author, channel } = message;
 
     const validateRes = validateBetAmount(bet);
-    if (validateRes !== true) {
-      // send error message if validation function doesn't return true
+    if (validateRes) {
```

Comment:
I was originally getting an error where according to type checks, `validateRes` might be false, so return `message.reply(validateRes)` won't work because `validateRes` must be a string (and not a boolean). Instead of `validateBetAmount` returning true when there is no error, an empty string is returned, so this condition checks that `validateRes` is non-empty.